### DISCUSSION TITLE
Ruby Serverless Runtime Conflicting with Jekyll Runtime

### DISCUSSION THREAD

**Discussion by DotIN13** [vercel/vercel#5994] [Help] [answered]
I have been working on a jekyll site with serverless functions, but whenever the serverless function also involves ruby environment, the function do not work. Whenever I visit `/api/function`, I get to download the `function.rb`, instead of getting the function response

The issue sticks in multiple scenarios, for example, when using ruby serverless functions with jekyll, and when building jekyll before building serverless functions to use built files in functions.

**Comment by andrealeone** [comment, marked as answer]
G'day DotIN13,

I incurred the same error working with Jekyll and the solution is quite straightforward.
In your `_config.yml` file, make sure to add the `/api` folder in the exclude list. This way, Jekyll won't copy it in `/public` during the build exposing the `function.rb` as file for download.

**Reply by DotIN13** [reply]
That's crazy straightforward! Thanks a bunch!
